triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/simulation/process/report/MetricsReporting.scala

```diff
@@ -91,4 +94,11 @@ private[simulation] object MetricsReporting {
         }
     }
   }
+
+  // We attempt to record just the completion summary prefix message (and not its detailed explanation)
+  private[this] def extractStatusMessage(status: GrpcStatus): String = {
+    val failureMessage = status.message.takeWhile('(' != _)
```

Comment:
For reporting, we only want the status summary tag and not the whole status description